http/src/main/resources/reference.conf

```diff
@@ -533,7 +533,6 @@ gke {
       {
         name = "cromwell-service"
         kind = "ClusterIP"
-        path = "/"
```

Comment:
I am not sure why this needs to removed now as it has been here for few months now. But if it fixes the issue then 👍